packages/sdk-browser/src/__types__/base.ts

```diff
@@ -91,6 +91,7 @@ export interface TurnkeySDKBrowserConfig {
   defaultOrganizationId: string;
   rpId?: string;
   serverSignUrl?: string;
+  iframeUrl?: string;
```

Comment:
because you're making a change here you'll need to generate a changeset for this PR so that we have a minor version change + changelog entry for sdk-browser